examples/data-sources/fmc_policy_assignments/data-source.tf

```diff
@@ -0,0 +1,3 @@
+data "fmc_policy_assignments" "example" {
+  id = "0050568A-2561-0ed3-0000-004294972836"
```

Comment:
Policy assignment does not have `id` assigned by FMC. This should be `policy_id`

Reply:
Removed data source and disabled generation of import.sh